webcamd.c

```diff
@@ -117,19 +118,28 @@ char	global_fw_prefix[128] = {"/boot/modules"};
 #define	v4b_errx(code, fmt, ...) do {			\
     syslog(LOG_ERR, "webcamd: " fmt "\n",##		\
 	__VA_ARGS__); 					\
+    pidfile_remove(local_pid);				\
```

Comment:
Do you think you need to have a non-NULL check in front of local_pid ?

Reply:
Not really needed IMHO, because the pidfile_remove() call will fail harmlessly in that case. But it wouldn't hurt, I can add it if you want. I wasn't even too sure about calling the pidfile_remove in v4b_errx() being needed, but it looked right. What do you think?

Reply:
I think you don't need to do pidfile_remove() here, because exit() will invoke the atexit() hook!

Reply:
Good point. Removed.